ChangeLog

```diff
@@ -1,5 +1,11 @@
 === rmw ChangeLog ===
 
+2021-05-19
+
+  * The config option 'purge_after' has been deprecated and replaced
```

Comment:
@Jammyjamjamman @sexybiggetje What do you think about this change? Any alternate suggestions for a replacement before I continue?